controllers/create_rolesbindings.go

```diff
@@ -50,7 +50,7 @@ func (r *SearchReconciler) createRoleBinding(ctx context.Context,
 			return &reconcile.Result{}, err
 		}
 		log.Info("Created clusterrolebinding" + rolebinding.Name)
-		log.V(2).Info("Created %s clusterrolebinding %+v", rolebinding)
+		log.V(2).Info("Created %s clusterrolebinding ", "clusterrolebinding", rolebinding)
```

Comment:
Should we remove %s here? @xrajesh

Reply:
yes thats a good idea , please push the change @SherinV